Tasks/AzureFileCopy/Utility.ps1

```diff
@@ -71,6 +71,22 @@ function Write-TaskSpecificTelemetry
     Write-Telemetry "$codeKey" "EB72CB01-A7E5-427B-A8A1-1B31CCAC8A43"
 }
 
+function Get-AzureCmdletsVersion
+{
+    $module = Get-Module AzureRM -ListAvailable
```

Comment:
We typically use AzureRM.Profile instead of AzureRM. That's what the Azure Handler used to do.